src/java.desktop/windows/classes/sun/awt/windows/WEmbeddedFrame.java

```diff
@@ -61,8 +59,7 @@ public class WEmbeddedFrame extends EmbeddedFrame {
     private boolean isEmbeddedInIE = false;
 
     @SuppressWarnings("removal")
-    private static String printScale = AccessController.doPrivileged(
-        new GetPropertyAction("sun.java2d.print.pluginscalefactor"));
+    private static String printScale = System.getProperty("sun.java2d.print.pluginscalefactor");
```

Comment:
You can get rid of the `@SuppressWarnings("removal")` here.

Also, this file has a now-unused import of `GetPropertyAction`.

Reply:
removed